.gitignore

```diff
@@ -14,3 +14,4 @@ examples/truffle-v4/types
 examples/truffle-v4/migrations
 examples/truffle-v5/types
 examples/truffle-v5/migrations
+.DS_Store
```

Comment:
this should go to your global gitignore file. otherwise we will end up with huge list of unrelevant OS files in this file.

Reply:
That makes sense. Removing this from gitignore file right away.